MAUI/TabView/Selection-Indicator-Customization.md

```diff
@@ -180,4 +180,21 @@ You can customize the corner radius of the selection indicator using the [Indica
 
 {% endtabs %} 
 
-![Selection Indicator Corner Radius.](images/IndicatorCornerRadius.png) 
\ No newline at end of file
+![Selection Indicator Corner Radius.](images/IndicatorCornerRadius.png) 
+
+## Indicator's Stroke Thickness
+You can customize the stroke thickness of the selection indicator using the [IndicatorStrokeThickness] property in the Tab View.
+
+{% tabs %}
+
+{% highlight xaml %}
+    <tabView:SfTabView IndicatorStrokeThickness ="7">
```

Comment:
check the code properly, end tag is missing

Reply:
I have updated the code using end tag properly.